veashi-contracts/.env.example

```
@@ -40,4 +40,19 @@ CCIP_ADAPTER_CHAIN_SELECTOR=16015286601757825753
 DEBRIDGE_ADAPTER_GATE_ADDRESS=0x43dE2d77BF8027e25dBD179B491e8d64f38398aA
 DEBRIDGE_REPORTER_FEE=100000000000000000
-DEBRIDGE_REPORTER_CHAIN_ID=100000013
\ No newline at end of file
+DEBRIDGE_REPORTER_CHAIN_ID=100000013
+
+# Axelar
+# Gateway + Gas Service on the reporter (source) chain
+AXELAR_REPORTER_GATEWAY=0xe432150cce91c13a887f7D836923d5597adD8E31
+AXELAR_REPORTER_GAS_SERVICE=0x2d5d7d31F671F86C782533cc367F14109a082712
+# Gateway on the adapter (destination) chain
+AXELAR_ADAPTER_GATEWAY=0xe432150cce91c13a887f7D836923d5597adD8E31
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AXELAR_ADAPTER_GATEWAY key should go before the AXELAR_REPORTER_GAS_SERVICE key

(UnorderedKey)